iconservice/prep/validator.py

```diff
@@ -60,6 +60,8 @@ def validate_prep_data(context: 'IconScoreContext', data: dict, set_prep: bool =
                 raise InvalidParamsException("Can not set empty data")
 
     for key in data:
+        if key == ConstantKeys.NODE_ADDRESS:
+            continue
```

Comment:
No need to check whether node_address is Address format, because type checking is already done by TypeConverter?

Reply:
Yes TypeConverter is Done, but that function is validate function.
That logic doesn't consider about address.

Reply:
Please move the node_address validation code to prep.validator package.